.github/dependabot.yml

```
@@ -1,7 +1,12 @@
 version: 2
 updates:
-- package-ecosystem: pip
-  directory: "/"
-  schedule:
-    interval: daily
-  open-pull-requests-limit: 10
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: daily
```

Is daily really needed?

```suggestion
      interval: weekly
```

---

As per comment on the other PR, the GitHub actions bump does not spam/bump that much.

---

I'd like to keep it like that for now to not add more load on Monday - Tuesday when the pip bumps come in. If we decide another schedule we can do that later.

---

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleday